src/Microsoft.ML.ModelBuilder.SearchSpace/Attribute/ChoiceAttribute.cs

```diff
@@ -0,0 +1,29 @@
+﻿// <copyright file="ChoiceAttribute.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
```

Comment:
The license we have at the top of the files is:
```
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.
```
We need to switch these to match.

Reply:
This applies to all the files.

Reply:
### Resolved